deploy/manifests/dev/us-east-2/tenant/storetheindex/dhfind/service-cluster-ip.yaml

```diff
@@ -0,0 +1,13 @@
+kind: Service
+apiVersion: v1
+metadata:
+  name: dhfind
```

Comment:
This name is already taken. service names must be unique in a namespace.

Reply:
Fixed